specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/storagecache.json

```diff
@@ -2586,6 +2533,16 @@
         }
       }
     },
+    "amlFilesystemArchiveInfo": {
+      "type":"object",
+      "description": "Information required to execute the archive operation",
+      "properties": {
+        "filesystemPath" : {
+          "type" : "string",
+          "description" : "The path to the filesystem to archive."
```

Comment:
I cannot figure out what this is from the description. The API docs are auto-generated from this, so it does matter.

Reply:
Also asked for suggestions below.  I think filesystemPath was suggested as input in a discussion.

Reply:
Updated field name to lustrePath.
Updated description to "Path relative to the root of the Lustre filesystem"